.github/workflows/ci-visibility-performance.yml

```diff
@@ -20,8 +20,7 @@ jobs:
       ROBOT_CI_GITHUB_PERSONAL_ACCESS_TOKEN: ${{ secrets.ROBOT_CI_GITHUB_PERSONAL_ACCESS_TOKEN }}
     steps:
       - uses: actions/checkout@v2
-      - uses: ./.github/actions/node/setup
+      - uses: ./.github/actions/node/18
       - run: yarn install
-      - uses: ./.github/actions/node/latest
```

Comment:
we seemed to have two setups? 🤔 

Also, no need to use `latest` (usually riskier), as the job is run elsewhere anyway, so the version of node should not matter